api/client/__init__.py

```diff
@@ -144,8 +144,8 @@ def get_data_series(self, **selection):
         return lib.get_data_series(self.access_token, self.api_host, **selection)
 
 
-    def get_data_points(self, **selection):
-        return lib.get_data_points(self.access_token, self.api_host, **selection)
+    def get_data_points(self, include_historical=True, **selection):
```

Comment:
Should `include_historical` should be one of the `selection`s, like `insert_null` or `show_revisions`? It just happens to be a client-side "selection" that'd get filtered out from the server request in `get_data_call_params()`, like how `unit_id` is

Reply:
commented below on `BatchClient. batch_async_get_data_points`